db/seeds.rb

```diff
@@ -160,6 +160,14 @@
   end
 end
 
+# Valuation matrix
+valuation_matrix = YAML.load_file("#{Rails.root}/db/valuation_matrix.yml")
```

Comment:
@steveyken , wondering if it would be better in a rake file? Thoughts

Reply:
You're thinking in the right direction! Rails actually does this for us in for `db/seeds.rb`. There is a rake task called `rake db:seed` which is very useful for setting up a database from scratch, esp in development mode. This ties in to that process so no need to change anything here.